temporalio/worker/workflow_instance.py

```diff
@@ -894,11 +899,15 @@ def workflow_start_local_activity(
                 retry_policy=retry_policy,
                 local_retry_threshold=local_retry_threshold,
                 cancellation_type=cancellation_type,
+                headers=None,
```

Comment:
Wouldn't it be easier for interceptor implementors to get an empty headers object here?

Reply:
Just empty dict, and remove the Optional type here.

Reply:
I say here but I mean all interceptor methods that accept headers

Reply:
On inbound, I make it an optional mapping because I was preserving the difference between absent headers and empty headers. But I guess it's not that important to preserve that difference.

Reply:
Done